# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: devtools/server/devtools-server.js
# Commit: 6eaa4d12b58c
# Full Hash: 6eaa4d12b58cb4762729fdf53511450926383db8
# Author: Nicolas Chevobbe <nchevobbe@mozilla.com>
# Date: 2020-10-15 09:40:06
# Regressor Bug: 1633712
# File Overlap Count: 1
# Description:
#   Bug 1633712 - [devtools] Create WorkerTargets as soon as possible. r=ochameau,devtools-backward-compat-reviewers.
#   
#   This patch adds support for dedicated worker targets in the Watcher actor.
#   Shared and Service workers are not handled yet.
#   
# ==============================================================================

diff -r 10f97f47c883 -r 6eaa4d12b58c devtools/server/devtools-server.js
--- a/devtools/server/devtools-server.js	Thu Oct 15 05:11:58 2020 +0000
+++ b/devtools/server/devtools-server.js	Thu Oct 15 05:17:52 2020 +0000
@@ -345,10 +345,10 @@
     return this._onConnection(transport, prefix, true);
   },
 
-  connectToParentWindowActor(devtoolsFrameActor, forwardingPrefix) {
+  connectToParentWindowActor(jsWindowChildActor, forwardingPrefix) {
     this._checkInit();
     const transport = new JsWindowActorTransport(
-      devtoolsFrameActor,
+      jsWindowChildActor,
       forwardingPrefix
     );
 